hazelcast-command-line/pom.xml

```diff
@@ -0,0 +1,78 @@
+<!--
+  ~ Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>hazelcast-command-line</name>
+    <artifactId>hazelcast-command-line</artifactId>
+    <packaging>jar</packaging>
```

Comment:
I don't think we want to move this as a whole module.

In particular, I think we want to take `com.hazelcast.commandline.HazelcastCommandLine` (and all code it uses, maybe inline into less classes if possible), move it to the `hazelcast` module, probably `com.hazelcast.core.server` package

We should think about a better name for the class with `hz` command - we already have `com.hazelcast.client.console.HazelcastCommandLine` for `hz-cli`

These 2 classes are very similar, I suppose they could be merged.
com.hazelcast.commandline.HazelcastMember
com.hazelcast.core.server.HazelcastMemberStarter

Reply:
> I don't think we want to move this as a whole module.

Why not as a module? A separate module improves encapsulation. It's easier to see what dependencies and resources are used or not. Also it improves parallelism of the builds.

> We should think about a better name for the class with hz command - we already have 
> com.hazelcast.client.console.HazelcastCommandLine for hz-cli

> We should think about a better name for the class with hz command - we already have com.hazelcast.client.console.HazelcastCommandLine for `hz-cli`

I left as it is to be backward compatible. 

> These 2 classes are very similar, I suppose they could be merged.
> com.hazelcast.commandline.HazelcastMember
> com.hazelcast.core.server.HazelcastMemberStarter

Yes, that was one of my follow-ups that needs more discussion on what features should be merged

Reply:
`HazlecastMember` was removed in https://github.com/hazelcast/hazelcast/pull/20262/commits/9a45e50cdd0c20dd973b5efa47034c7f99dc2993

`hazelcast-command-line` was merged with `hazelcast-distribution` in https://github.com/hazelcast/hazelcast/pull/20262/commits/68b1fe7757ee5628255aeb3a1f30f178a9b678d5